src/style.css

```diff
@@ -1,3 +1,88 @@
+@import url("https://fonts.googleapis.com/css2?family=Karla:ital,wght@0,200;0,300;0,400;0,500;0,600;0,700;0,800;1,200;1,300;1,400;1,500;1,600;1,700;1,800&family=Spectral:ital,wght@0,200;0,300;0,400;1,200;1,300;1,400&display=swap");
+
+body {
+  margin: 0;
+  font-family: "Karla", sans-serif;
+  background: #ecf0f3;
+  display: grid;
+  place-items: center;
+}
+
+.modal-container {
+  display: flex;
+  justify-content: center;
+  position: absolute;
+  top: 0;
+  left: 0;
+  bottom: 0;
+  width: 100vw;
+  background-color: rgba(0, 0, 0, 0.5);
+  padding: 20px 10px;
+}
+
+.modal {
+  width: 80%;
+  height: 100%;
+  background-color: #fff;
+  border: 1px solid #000;
+}
+
+.modal-content {
+  width: 76%;
+  padding: 15px 30px;
+  position: absolute;
+}
+
+.close {
+  position: relative;
+  top: 20px;
+  left: 95%;
+}
+
+.modal-body {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 15px;
+  width: 80%;
+  margin: 0 auto;
+}
+
+.modal-body img {
+  height: 250px;
+  width: 100%;
+  object-fit: contain;
+}
+
+.form-content {
+  display: flex;
+  flex-direction: column;
+  width: 80%;
+  padding-left: 20px;
+  gap: 10px;
+}
+
+.result {
+  font-size: 1em;
+  margin-right: 210px;
+}
+
+.result h4 {
+  font-size: 1.5em;
+  margin-bottom: 8px;
+}
+
+.modal-title {
+  font-size: 1em;
+}
+
+.comment-head {
+  font-size: 1.5em;
+  margin-bottom: 0;
+  margin-top: 8px;
+  margin-right: 260px;
+}
```

Comment:
:ballot_box_with_check: Kindly put these style after your general style and organize it according to your html file structure in order to make your css file more readable and understandable.